code/datums/achievements/misc_achievements.dm

```diff
@@ -111,3 +111,9 @@
 	name = "HA, GOTTEM"
 	desc = "Made you look!"
 	database_id = MEDAL_GOTTEM
+
+/datum/award/achievement/misc/ascension
+	name = "Ascension"
```

Comment:
might want to make it clear that this refers to a His Grace ascension, not a heretic one.

Reply:
its obvious